qark/decompiler/decompiler.py

```diff
@@ -121,15 +143,20 @@ def run_apktool(self):
 
         custom_apktool_command = APK_TOOL_COMMAND.format(apktool_path=apktool_path,
                                                          path_to_apk=self.path_to_apk,
-                                                         build_directory=self.build_directory)
+                                                         build_directory=os.path.join(self.build_directory, "apktool"))
         try:
             subprocess.call(shlex.split(custom_apktool_command))
         except Exception:
             log.exception("Failed to run APKTool with command: %s", custom_apktool_command)
             raise SystemExit("Failed to run APKTool")
 
-        if os.path.isdir(os.path.join(self.build_directory, "original")):
-            shutil.rmtree(os.path.join(self.build_directory, "original"))
+        # copy valid XML file to correct location
+        shutil.copy(os.path.join(self.build_directory, "apktool", "AndroidManifest.xml"),
```

Comment:
You're deleting the directory right below here, you might as well just `shutil.move` the manifest

Reply:
Will change. The documentation confused me for `shutil.move` since the destination IS on the current file system:
>If the destination is on the current filesystem, then os.rename() is used. Otherwise, src is copied (using shutil.copy2()) to dst and then removed.